src/index.ts

```diff
@@ -61,6 +93,16 @@ export class GraphQLClient {
   }
 }
 
+export async function rawRequest<T extends any>(
+  url: string,
+  query: string,
+  variables?: Variables,
+): Promise<T> {
```

Comment:
Please adjust to `Promise<{ data?: T, extensions?: any, errors?: GraphQLError }>`